tunelinux/multipath.go

```diff
@@ -328,6 +329,23 @@ func SetMultipathRecommendations() (err error) {
 			return err
 		}
 	}
+	// Get current recommendations
+	recommendations, err := GetMultipathRecommendations()
+	if err != nil {
+		return err
+	}
+	if len(recommendations) == 0 {
+		log.Info("no recommendations found for multipath.conf settings")
```

Comment:
Missed this one - can you make this a warning level log

Reply:
Done. Log level changed to Warning.